subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -171,7 +171,9 @@
                             <li><a class="nav-dropdown" data-toggle="collapse" href="#managing-dependencies" aria-expanded="false" aria-controls="managing-dependencies">Managing Dependencies</a>
                                 <ul id="managing-dependencies">
                                     <li><a href="/userguide/artifact_dependencies_tutorial.html">Dependency Management Basics</a></li>
-                                    <li><a href="/userguide/dependency_management.html">Advanced Dependency Management</a></li></ul></li>
+                                    <li><a href="/userguide/dependency_management.html">Advanced Dependency Management</a></li>
+                                    <li><a href="/userguide/dependency_types.html">Dependency Types</a></li>
```

Comment:
I think perhaps "Advanced Dependency Management" can go last in the list here. 

Separately, we're using Title Case in the menu but Sentence case on the page. We should make page titles Title Case for consistency.

Reply:
> I think perhaps "Advanced Dependency Management" can go last in the list here.

Can you explain why? I think of "Dependency Types" and "Repository Types" as glossaries - something you want to reference but not read in order.

> Separately, we're using Title Case in the menu but Sentence case on the page. We should make page titles Title Case for consistency.

I feel like we didn't make a decision on this yet based on the internal discussion. At the least for now we should make it consistent. PR: https://github.com/gradle/gradle/pull/3967

Reply:
Further reordering will be tackled with this issue: https://github.com/gradle/dotorg-docs/issues/133. For now we'll leave the left nav bar items ordered as-is.